### DISCUSSION TITLE
Argon2 Upstream

### DISCUSSION THREAD

**Discussion by crespire** [dani-garcia/vaultwarden#3246] [General]
Looks like upstream has Agron2 available for webvault, but not quite yet for the clients:

https://www.reddit.com/r/Bitwarden/comments/112onwn/argon2_support_please_wait_until_202320_is/

Looking forward to the update to vw!

**Comment by ghost** [comment]
> Looks like upstream has Agron2 available for webvault, but not quite yet for the clients:
> 
> https://www.reddit.com/r/Bitwarden/comments/112onwn/argon2_support_please_wait_until_202320_is/
> 
> Looking forward to the update to vw!

2023.2.0 is out on their Github in fact

**Comment by ghost** [comment]
how and also it's a dupplicate https://github.com/dani-garcia/vaultwarden/discussions/3171
